doc/sphinx/source/projects/ad7124_console/ad7124_console.rst

```diff
@@ -0,0 +1,181 @@
+
+===============================
+Power Supply and USB Connection
+===============================
+
+A 7V to 9V DC supply (barrel jack, center pin positive) is required to power the EVAL-AD7124-8SDZ evaluation board.
+SDP-K1 is powered through USB connection to PC. SDP-K1 acts as a Serial device when connected to PC, which creates 
+a COM Port to connect to Serial Terminals like Teraterm, Putty, etc. The COM port assigned to a device can be seen 
+through the device manager for windows based OS.
+
+   .. image:: /source/projects/ad7124_console/com_port_sdp-k1.jpg
+      :width: 300
+
+=============
+Analog Inputs
+=============
+
+The screw terminal connections to J6 and J11 can be used to connect appropriate analog input signals to provide test stimulus to the AD7124.
+
+The AD7124 device can be software configured to interface with external devices (e.g. sensors). The AD7124 Firmware 
+example provides such 2 configurations (selected through console UI) as described below:
+
+**Configuration A:**
+
+AIN0/AIN1 are used for channel 0, simple voltage measurement
+
+**Configuration B:**
+
+AIN2/AIN3 go to the A2 thermocouple connector on the evaluation board, and are captured on channel 0. 
+This uses an internal reference and has a bias voltage enabled on AIN2. A suitable thermocouple should be
+connected to A2 for this measurement. AIN4/AIN5 are an RTD1000 measurement on channel 1. Excitation is provided
+from AIN1 for this. This requires an external RTD and reference resistor connected as show in the figure below.
+
+
+   .. image:: /source/projects/ad7124_console/ad7124_rtd_thermocouple_connections.png
+      :width: 200
+
+Temperature sensing application using AD7124 can be found `here <https://www.analog.com/media/en/reference-design-documentation/reference-designs/CN0383.pdf>`_
+
+====================
+AD7124 Mbed Firmware
+====================
+This section briefs on the usage of MBED firmware. This also explains the steps to compile and build the application using
+mbed and make based build.
+
+   .. image:: /source/projects/ad7124_console/ad7124_software_sequence.jpg
+      :width: 700
+
+The software execution sequence for the AD7124 Firmware Example is shown below. This is a blocking application as it waits for 
+user input over serial interface (UART). The input is scanned and processed through 'adi console libraries'. The menu functionality
+is executed from ad7124_conole_app.c file. The application layer talks with No-OS layer for device registers and data access. The No-OS
+layer interfaces with Platform Drivers layer for accessing low level peripherals. As name suggests, this layer is platform dependent. 
+AD7124 firmware uses Mbed libraries within Platform Drivers layer.
+
+.. Project Build Section:
+    
+.. include:: /source/build/project_build.rst
+
+===========
+Quick Start
+===========
+
+If you have some familiarity with the Mbed platform, the following is a basic list of steps required to start running the code, 
```

Comment:
I think Quick Start needs to be modified a bit as building and flashing the binary is already covered in the Build section.
Lines 128 to 137 are redundant and there isn't a Downloads section as mentioned in line 133.